.github/workflows/issue-opened.yml

```
@@ -61,7 +61,7 @@ jobs:
               // for now check for comma or space separated version.
               const versions = electronVersion.split(/, | /);
               for (const version of versions) {
-                const major = semver.parse(version)?.major;
+                const major = semver.coerce(version)?.major;
```

```suggestion
                const major = semver.coerce(version, { loose: true })?.major;
```

I believe we can add this to make version parsing even more forgiving.

It causes `semver` to allow certain non-critical mistakes that `semver` can auto-correct.

You can check [this file](https://github.com/npm/node-semver/blob/145c554b8c7b7ecfcb451153ad18bdb2f24ad10d/internal/re.js) for the differences in regular expressions with and without the `loose` option.